source/install/install-kubernetes-custom.rst

```diff
@@ -0,0 +1,62 @@
+.. _install-kubernetes-custom:
```

Comment:
This section is in progress and was included in this PR for discussion. Currently adding a section on replication, as well as storage requirements using the content from https://github.com/mattermost/mattermost-operator/blob/master/pkg/apis/mattermost/v1alpha1/clusterinstallation_sizes.go. 

It's slow-going so if there's a brain dump somewhere that outlines the basics of replication and storage (in conjunction with the info provided) that'd be great.

Reply:
Discussed this with @wiersgallak and considering removing it - however it may be useful to customers restoring their db and needing more custom configs (and understanding replication).

Reply:
If this is holding up the publish, recommend we start another PR (or open an issue to track) with this content to be worked on after this one is approved.

Reply:
Agreed. I'll move it out.